niworkflows/anat/mni.py

```diff
@@ -138,8 +156,43 @@ def _config_ants(self, ants_settings):
         if self.inputs.testing:
             resolution = 2
 
-        norm.inputs.fixed_image = op.join(
+        self.norm.inputs.fixed_image = op.join(
             mni_template, '%dmm_%s.nii.gz' % (resolution, self.inputs.reference))
-        norm.inputs.fixed_image_mask = op.join(mni_template,
-                                               '%dmm_brainmask.nii.gz' % resolution)
-        return norm
+        self.norm.inputs.fixed_image_mask = op.join(
+            mni_template, '%dmm_brainmask.nii.gz' % resolution)
+
+
+    def _generate_report(self):
+        """ Generates the visual report """
+        NIWORKFLOWS_LOG.debug('Generating visual report')
+
+        def _plot_xyz(anat_img, plot_params, div_id):
+            """
+
+            """
+            NIWORKFLOWS_LOG.info('Plotting %s', str(anat_img))
+            svgs = []
+
+            for mode, cuts in list(DEFAULT_CUTS.items()):
+                plot_params['display_mode'] = mode
+                plot_params['cut_coords'] = cuts
+                image = plot_anat(anat_img, **plot_params)
+                svgs.append('<div id="{0}-{1}" class="{0}">{2}</div>'.format(
+                    div_id, mode, report.as_svg(image)))
+                image.close()
+            return '\n'.join(svgs)
+
+        plot_params = {'cut_coords': 3}
+        report.save_html(
+            template='overlay_3d_report.tpl',
+            report_file_name=self._results['html_report'],
+            unique_string='antsRegistration',
```

Comment:
> unique_string: Yeah, I was reluctant to add another dependency to the code. I would prefer to pick up the identifier of the nipype interface or so.

I had thought about this, but I don't think the unique node name is accessible from inside `_generate_reports()`. Can you think of a way to get it? uuid is in python's standard library and it's not a very big dependency so I think it's reasonable to use it.

re: ids, check out my pr here: https://github.com/poldracklab/niworkflows/pull/19